lib/VirtualObjectIdManager.cpp

```diff
@@ -259,6 +259,55 @@ sai_object_id_t VirtualObjectIdManager::allocateNewObjectId(
     return objectId;
 }
 
+void VirtualObjectIdManager::allocateNewObjectIds(
+        _In_ sai_object_id_t switchId,
+        _In_ sai_object_type_t* objectTypes,
+        _Out_ sai_object_id_t* oids,
```

Comment:
oids should be lats parameter, and count should be before objectTypes parameter

Reply:
done